workloads/kube-burner/README.md

```diff
@@ -17,25 +17,32 @@ All scripts can be tweaked with the following environment variables:
 |------------------|-------------------------------------|---------|
 | **OPERATOR_REPO**              | benchmark-operator repo                     | https://github.com/cloud-bulldozer/benchmark-operator.git      |
 | **OPERATOR_BRANCH**              | benchmark-operator branch                     | master      |
-| **QPS**              | Queries/sec                     | 20      |
-| **BURST**            | Burst queries                   | 20      |
-| **ES_SERVER**        | Elastic search endpoint         | https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443 (You can disable indexing with `export ES_SERVER=""`)|
+| **INDEXING**         | enable/disable indexing         | true    |
+| **ES_SERVER**        | Elastic search endpoint         | https://search-perfscale-dev-chmf5l4sh66lvxbnadi4bznl3a.us-west-2.es.amazonaws.com:443|
 | **ES_INDEX**         | Elastic search index            | ripsaw-kube-burner|
 | **PROM_URL**         | Prometheus endpoint         | https://prometheus-k8s.openshift-monitoring.svc.cluster.local:9091|
-| **JOB_TIMEOUT**      | kube-burner's job timeout, in seconds      | 17500 |
-| **POD_READY_TIMEOUT**| Timeout for kube-burner and benchmark-operator pods to be running | 1200 |
-| **WORKLOAD_NODE**    | Workload node selector          | {"node-role.kubernetes.io/worker": ""} |
-| **CERBERUS_URL**     | URL to check the health of the cluster using [Cerberus](https://github.com/cloud-bulldozer/cerberus) | "" (don't check)|
+| **METADATA_COLLECTION**    | Enable metadata collection | true (If indexing is disabled metadata collection will be also disabled) |
+| **JOB_TIMEOUT**      | kube-burner's job timeout, in seconds      | 14400 (4 hours) |
+| **POD_READY_TIMEOUT**| Timeout for kube-burner and benchmark-operator pods to be running | 180 |
+| **NODE_SELECTOR**    | The kube-burner pod deployed by benchmark-operator will use this node selector          | {node-role.kubernetes.io/worker: } |
+| **QPS**              | Queries/sec                     | 20      |
+| **BURST**            | Burst queries                   | 20      |
```

Comment:
What's the difference between `QPS` and `BURST`?

Reply:
I've updated the description in the doc